config/initializers/active_metrics.rb

```diff
@@ -0,0 +1 @@
+ActiveMetrics::Base.establish_connection(:adapter => "miq_postgres", :database => "metrics_spike")
```

Comment:
If this gets merged the database will need to change...I'm guessing this would end up being `Rails.database_configuration.database` (or however you get it)